dts/bindings/clock/nordic,nrf-hfxo.yaml

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nordic nRF high-frequency crystal oscillator
+
+compatible: "nordic,nrf-hfxo"
+
+include: [fixed-clock.yaml]
+
+properties:
+  clock-frequency:
```

Comment:
How about `crystal-frequency`? So it's not confused with frequencies generated by CLOCK.